gossip/minimal/goshawk.go

```diff
@@ -109,21 +125,15 @@ func NewGoshawk(ctx context.Context, cfg *configpb.GoshawkConfig, hc *http.Clien
 		}
 		seenNames[base.Name] = true
 
-		if _, ok := origins[base.URL]; ok {
+		if _, ok := hawk.origins[base.URL]; ok {
```

Comment:
Use of ok name leads to if ok { error }, which reads a bit odd. Might be a go religious thing not to be questioned though.

Reply:
Renamed `ok` to `exists` for these dup-detection cases.